### PR TITLE
Add success stories

### PR DESCRIPTION
**Pull request by trabaldini**
This adds case studies for DD-WRT, Timify and trainline (unclear whether we'll be allowed to publish trainline):

@EmanuelaTrabaldoLena: can you check please whether all the texts are up to date with the latest texts in basecamp? Can you also add `alt` tags to the images in the success stories (see output of failing build: https://travis-ci.org/simplabs/simplabs.github.io/builds/277406311?utm_source=github_status&utm_medium=notification)?

localhost_4000_cases_dd-wrt ipad pro
localhost_4000_cases_timify ipad pro
localhost_4000_cases_trainline ipad pro

### CODE REVIEW

**Comment by marcoow** [comment]
>  made a page for the list of case studies 

I don't think we need the list page at all for now.

File: _config.yml [hunk 1]
```diff
@@ -53,4 +53,4 @@ defaults:
         twitter: /images/apple-144.png
         facebook: /images/apple-144.png
         height: 144
-        width: 144
+        width: 144
```
Comment (marcoow) [inline _config.yml]: These changes seem to be unrelated?

File: casestudies/trainline.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <h6>ruby on rails, custom engineering, consulting</h6>
+      <div class="row content-section" id="wrapper">
+        <div class="col-md-6 col-sm-12" id="image">
+          <img src="/images/trainline-desktop.png" class="img-responsive">
+        </div>
+        <div class="col-md-6 col-sm-12" id="text">
+          <h4>trainline</h4>
+          <p>trainline is the leading booking site for European train tickets. They are selling tickets to customers worldwide, enabling more than 125,000 journeys every single day in and across 24 countries, covering 86 different carriers.</p>
+          <p>trainline is also an early Ember.js adopter, having started to build their desktop website with Ember.js' predecessor SproutCore as early as 2010 and having stuck with the ecosystem ever since.</p>
+        </div>
```
Comment (marcoow) [inline casestudies/trainline.html]: Somehow the text doesn't wrap to the next line here on narrow viewports:

bildschirmfoto 2017-08-04 um 16 17 51

File: casestudies/trainline.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+          <p>In addition to supporting trainline with engineering resources and expertise we were also able to help improve the engineering process, establish and solidify a sprint based flow, resulting in improved planability and developer insight.</p>
+          <p>Ember.js proved to be a great to choice for this kind of complex application. With its focus on conventions it allows for decent development pace while still ensuring long term stability and maintainability. Technologies like FastBoot and engines allow building large and complex applications for mobile devices without having to sacrifice performance and usability.</p>
+          <p>As of June 2017, we are still an active part of the engineering team behind both the desktop and mobile sites, leading optimization efforts as well as driving development efforts for additional features. </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row content-button">
+    <a href="/casestudies" class="custom-button case-button">All case studies</a>
+  </div>
```
Comment (marcoow) [inline casestudies/trainline.html]: I don't think we need this button at all. Instead I'd show links to 2 other case studies (as we do on the clients page) under each study.

**Review by marcoow — COMMENTED** [review]
Looks nice!

Sorry I forgot to say that but the idea was that we'd be using markdown for the success stories as we do e.g. for the blog posts. That makes maintaining the content and applying consistent styling etc. much easier.

There's the `cases` folder already with a sample story. Can you try whether you can integrate that with this somehow?

File: _config.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@ navigation:
     url: /clients/
   - title: Blog
     url: /blog/
-
+
```
Comment (marcoow) [inline _config.yml]: 👀

File: _layouts/case.html [hunk 5]
```diff
@@ -5,29 +5,20 @@
     {% include nav.html %}
 
     <div id="main-content">
-      <div class="primary-banner">
+      <div class="primary-banner case">
```
Comment (marcoow) [inline _layouts/case.html]: Is this necessary? The primary banner should look the same as on other pages, no?

File: _layouts/case.html [hunk 6]
```diff
@@ -5,29 +5,20 @@
...
-      <div class="container" id="main-section">
-        <div class="row">
-          <div class="col-md-10 col-md-offset-1 col-sm-12">
-            <div class="row content-section article">
-              <div class="col-sm-12">
-                {{page.content}}
-              </div>
-            </div>
-          </div>
+      <div class="row content-section article">
```
Comment (marcoow) [inline _layouts/case.html]: Might be better to keep the `main-section` div? Also I don't think we should use the `article` class here as that's for blog posts I think?

**Review by marcoow — COMMENTED** [review]
This is coming together nicely. Few points:

* the image captions should be actual text and not be part of the images
* The sections with images and text side-by-side should wrap so that the image and text are vertically stacked on small devices: 
bildschirmfoto 2017-09-14 um 20 42 05
* the quotes need some horizontal padding: 
bildschirmfoto 2017-09-14 um 20 43 00

File: _layouts/case.html (hunk 5 above)
Comment (trabaldini) [reply _layouts/case.html]: If there ist just a h2-title without a p-text, it looks better when it's at the center of the banner. If we write some text under the title it's not necessary.

Comment (marcoow) [reply _layouts/case.html]: I see - I'll come up with some text to add here.

**Review by marcoow — COMMENTED** [review]
Cool! I will insert the missing texts etc.

File: cases/_posts/2017-1-1-trainline.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  </div>
+  <div class="col-xs-6 col-xs-offset-3 col-sm-4 col-sm-offset-1 col-md-3 col-md-offset-1">
+    <img src="/images/cases/trainline/mobile.png" class="img-responsive">
+  </div>
+</div>
+
+<div class="row content-section">
+  <div class="col-xs-12 col-sm-6 col-sm-push-6 col-md-5 col-md-offset-1 col-md-push-5">
+    <p>In addition to supporting trainline with engineering resources and expertise we were also able to help improve the engineering process, establishing and solidifying a sprint based flow, resulting in improved plan-ability and developer insight.</p>
+    <p>Ember proved to be a great choice for this kind of complex application. With its focus on conventions it allows for decent development pace while still ensuring long term stability and maintainability. <strong>Technologies like FastBoot and engines allow building large and complex applications for mobile devices without having to sacrifice performance and usability</strong>.</p>
```
Comment (pangratz) [inline cases/_posts/2017-1-1-trainline.md]: First time `Ember` is used instead of `Ember.js` on this page. I guess for the sake of consistency we should stick with `Ember.js` here...

File: cases/_posts/2017-1-1-trainline.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+title: "Trainline"
+excerpt: "a mobile train ticket counter"
+permalink: /cases/trainline
+---
+
+<div class="row content-section">
+  <div class="col-xs-12 col-sm-6 col-sm-push-6 col-md-5 col-md-push-5 col-md-offset-1">
+    <h3>trainline</h3>
+    <p>trainline is the <strong>leading booking site for european train tickets</strong>. They are selling tickets to customers worldwide, enabling more than 125,000 journeys every single day in and across 24 countries, covering 86 different carriers.</p>
+    <p>trainline is also an early Ember.js adopter, having started to build their desktop website with Ember.js' predecessor SproutCore as early as 2010 and sticked with the ecosystem ever since.</p>
```
Comment (niklaslong) [inline cases/_posts/2017-1-1-trainline.md]: Just a small thing, the past tense of `to stick` is `stuck`. I also think this might read slightly better: `Trainline is also an early Ember.js adopter. They started building their desktop website with Ember.js' predecessor, SproutCore, as early as 2010 and have stuck with the ecosystem ever since.`

**Review by pangratz — APPROVED** [review]

**Review by niklaslong — APPROVED** [review]
It looks great!

File: cases/_posts/2017-1-1-trainline.md (hunk 8 above)
Comment (marcoow) [reply cases/_posts/2017-1-1-trainline.md]: 👍 fixed

**Review by oliverbarnes — APPROVED** [review]

File: cases/_posts/2017-1-1-trainline.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+section: Cases
+title: "Trainline"
+excerpt: "a mobile train ticket counter"
+permalink: /cases/trainline
+---
+
+<div class="row content-section">
+  <div class="col-xs-12 col-sm-6 col-sm-push-6 col-md-5 col-md-push-5 col-md-offset-1">
+    <h3>trainline</h3>
+    <p>trainline is the <strong>leading booking site for european train tickets</strong>. They are selling tickets to customers worldwide, enabling more than 125,000 journeys every single day in and across 24 countries, covering 86 different carriers.</p>
```
Comment (jayjayjpg) [inline cases/_posts/2017-1-1-trainline.md]: I think this should be written as
> trainline is the <strong>leading booking site for European train tickets</strong>. [...]

as [adjectives derived from proper nouns ("Eigennamen" = nouns describing locations, languages, people, organizations, etc.) are commonly capitalized in English](https://en.wikipedia.org/wiki/Capitalization#Adjectives), also e.g. 

- a Christian church
- the Canadian syrup 
- a Shakespearean sonnet

File: cases/_posts/2017-1-1-trainline.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    <p>simplabs's team merged with trainline's internal engineering team and helped them build and extend a mobile site, accompanying the existing dektop app.</p>
```
Comment (jayjayjpg) [inline cases/_posts/2017-1-1-trainline.md]: I think it should be

> accompanying the existing ~~dektop~~ desktop app.

Comment (jayjayjpg) [inline cases/_posts/2017-1-1-trainline.md]: Also to me it feels that the company name `simplabs` ending in a `s` and indicating that its a plural form (in contrast to `simplab`) should omit the additional `s` in its possessive form

> ~~simplabs's~~ simplabs'

Yet, since it is an [organization name we are probably free to use either as mentioned in this similar question here](https://english.stackexchange.com/questions/232796/guidelines-for-adding-s-after-name-of-company-ending-in-s) and [here](https://owl.english.purdue.edu/owl/resource/621/1/). I only believe it would be good to be consistent throughout the text

File: cases/_posts/2017-1-2-timify.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    <img src="/images/cases/timify/daily-view.png" class="img-responsive">
+  </div>
+</div>
+
+<div class="secondary-banner quote-banner">
+  <div class="container">
+    <div class="row content-section">
+      <div class="col-xs-12 col-md-10 col-md-offset-1">
+        <blockquote>
+          <p>simplabs' highly skilled engineers delivered a solid and well architected foundation for our web app. Working with them also helped us establish best practices and a lean process internally. Working with them was a pleasure.</p>
```
Comment (jayjayjpg) [inline cases/_posts/2017-1-2-timify.md:25]: I think this form `simplabs'` feels more consistent, but this should be updated if we decide for the other form `simplabs's`

File: cases/_posts/2017-1-3-dd-wrt.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+    <img src="/images/cases/dd-wrt/overview.png" class="img-responsive">
+  </div>
+</div>
+
+<div class="secondary-banner quote-banner">
+  <div class="container">
+    <div class="row content-section">
+      <div class="col-xs-12 col-md-10 col-md-offset-1">
+        <blockquote>
+          <p>simplabs’ expertise in developing Ember.js based applications helped us significantly to define and develop the new DD-WRT NXT user experience. It was impressive to see in what short time such a complex application leveraging a customer backend interface could be implemented. simplabs' work did provide the new foundation for our UI and our in-house development team greatly benefitted from their know how.</p>
```
Comment (jayjayjpg) [inline cases/_posts/2017-1-3-dd-wrt.md:25]: I just realize that we mostly went for `simplabs'` already, so let's stick to that :thumbsup:

**Review by jayjayjpg — COMMENTED** [review]
This looks great, just a few grammar nitpicks from my side 👍

File: cases/_posts/2017-1-1-trainline.md (hunk 9 above)
Comment (marcoow) [reply cases/_posts/2017-1-1-trainline.md]: 👍 😀

File: cases/_posts/2017-1-1-trainline.md (hunk 10 above)
Comment (marcoow) [reply cases/_posts/2017-1-1-trainline.md]: yes, right of course

Comment (marcoow) [reply cases/_posts/2017-1-1-trainline.md]: 🤦‍♂️

**Review by jayjayjpg — APPROVED** [review]

**Review by ninaarndt — APPROVED** [review]
